libenkf/src/rng_config.c

```diff
@@ -101,6 +130,7 @@ rng_config_type * rng_config_alloc(const config_content_type * config_content) {
 void rng_config_free( rng_config_type * rng) {
   util_safe_free( rng->seed_load_file );
   util_safe_free( rng->seed_store_file );
+  util_safe_free( rng->random_seed );
```

Comment:
The `util_safe_free( )` originates in a misunderstanding - it is safe to call `free( NULL )`.

Reply:
Aha, I never really understood the reason for `util_safe_free` to exist. If you agree, we could deprecate it then?